### ISSUE TITLE
Regression when compiled using Zig/Clang. test_diff_[correctout_geom.txt]

### ISSUE THREAD

**Issue by JamesParrott** [fiftysevendegreesofrad/sdna_plus#64]
```
E               AssertionError: Error on line num i: 2.  This line (and up to the 9 previous ones):
E                 Expected: 'Shapefile  tinyout_1_10_cont_geodesics.shp ====================================',
E
E                  Actual: 'Shapefile  tinyout_1_10_cont_.shp ===================================='
```

**Comment by JamesParrott** [comment]
Will be fixed by https://github.com/JamesParrott/sdna_plus/commit/02ff9817a2be9ef19d897288da8eb1f4ee6b02e6

**Comment by JamesParrott** [comment]
This was due to a coding error (when changing class variable scopes for the prefixes and postfixes, to fix something else) that has now been corrected. Fixed by #75
